diff --git a/dist/chunks/codegen-tasks.cjs b/dist/chunks/codegen-tasks.cjs
index 3f973daa8faae4c8eff0b56e0bd3f6a709fd2206..f0d5fa56775da992585490f575e4818e56cad543 100644
--- a/dist/chunks/codegen-tasks.cjs
+++ b/dist/chunks/codegen-tasks.cjs
@@ -89,6 +89,7 @@ const codegenTasks = () => __async(void 0, null, function* () {
       nextDir,
       "../..",
       "client",
+      "react-client-callbacks",
       "on-recoverable-error.js"
     );
     const readFile = yield fs__default.readFile(nextClientIndex);
diff --git a/dist/chunks/codegen-tasks.mjs b/dist/chunks/codegen-tasks.mjs
index e54d850e406b3ca6b08ae67f8ea438a916603de0..1bd770a986365852b2d6e6e2b50b886815b178e8 100644
--- a/dist/chunks/codegen-tasks.mjs
+++ b/dist/chunks/codegen-tasks.mjs
@@ -80,6 +80,7 @@ const codegenTasks = () => __async(void 0, null, function* () {
       nextDir,
       "../..",
       "client",
+      "react-client-callbacks",
       "on-recoverable-error.js"
     );
     const readFile = yield fs.readFile(nextClientIndex);
